governance/program/tests/program_test/mod.rs

```diff
@@ -3143,4 +3143,42 @@ impl GovernanceProgramTest {
 
         Ok(max_voter_weight_record_cookie)
     }
+
+    #[allow(dead_code)]
+    pub async fn with_complete_proposal(
```

Comment:
nit: The function doesn't return any cookie (it's imperative and doesn't create any account state) and hence should be named `complete_proposal`